### PR TITLE
Reorganize feature page to clearify stage sections.

### LINKED ISSUE #2969: The Development Stages header shouldn't be above Metadata
The Development stages header is currently right above the Metadata section which is not a stage itself. We should ensure that the Development stages header comes right before the actual stages sections.

image

### PR DESCRIPTION
**Pull request by jrobbins**
This should resolve #2969.

In this PR:
* Define a set of CSS styles that is shared between chromedash-feature-page and chromedash-feature-detail.
* Move the metadata section just above the heading "Development stages".
* Add a "stage" CSS class and change the CSS selector used find the stage accordion sections to look for that class.  This makes the expand-all / collapse-all link work for just the stage sections.
* Create a new "History" heading in the overview box:
    * add an entry created date
    * move the link to the comments & activity page here

### CODE REVIEW

**Comment by jrobbins** [comment]
Screenshot showing changes
image

File: client-src/elements/chromedash-feature-detail.js [hunk 1]
```diff
@@ -466,7 +467,7 @@ class ChromedashFeatureDetail extends LitElement {
         @sl-after-show=${this.updateCollapsed}
         @sl-after-hide=${this.updateCollapsed}
         ?open=${isActive || defaultOpen}
-        class=${isActive ? 'active' : ''}
+        class="${isActive ? 'active' : ''} ${isStage ? 'stage' : ''}"
```
Comment (KyleJu) [inline client-src/elements/chromedash-feature-detail.js:470]: How would it look like if it is both active and stage? class="activestage"?

**Comment by KyleJu** [comment]
> Screenshot showing changes image

Could perhaps add a bit vertical space between "all comments & activity" and  "last updated..."

File: client-src/elements/chromedash-feature-detail.js (hunk 1 above)
Comment (jrobbins) [reply client-src/elements/chromedash-feature-detail.js:470]: It has whitespace between the expressions, so it becomes `class="active stage"` which works.  That also means that that space is always there, but it does hurt anything.

**Comment by jrobbins** [comment]
I added some margin between the paragraphs.

**Review by KyleJu — APPROVED** [review]

**Pull request merged**